src/components/dashboard/FaceVerification/components/DuplicateFoundError.js

```diff
@@ -77,7 +77,7 @@ const getStylesFromProps = ({ theme }) => {
     halfIllustration: {
       marginTop: isMobileOnly ? getDesignRelativeHeight(32) : 0,
       marginBottom: isMobileOnly ? getDesignRelativeHeight(40) : 0,
-      width: getDesignRelativeWidth(100, false),
+      width: getDesignRelativeWidth(isBrowser ? 200 : 100),
```

Comment:
why 200 and 100? browser is not twice the width of mobile.
what should the width be according to the original design?
@LiavGut

Reply:
is getDesignRelativeWidth souldn't adjust the value passed according to the device screen size ?
probably it doesn't supports a case when we're running on desktop ? probably it's an issue and we should update getDesignRelativeWidth to support desktop ? to detect which are dimensions of the viewport RNW app running in and use them as screen size